### ISSUE TITLE
Change alt text for sdg image in access the data md file

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#3093]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will change the alt text for the access the data project's sustainability development goal image on [citizen engagement page](https://www.hackforla.org/citizen-engagement).

### Details
Designers and PMs identified that some alt texts in the Citizen Engagement page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] For the file `_projects/access-the-data.md`, change [line 111](https://github.com/hackforla/website/blob/f2ce8983e77a9b80d455e3677b1cf323092c4f8d/_projects/access-the-data.md?plain=1#L111) for the alt text:

from
```
sdg-image-alt: peace justice bottom sustainable dev goal
```
to
```
sdg-image-alt: '16: peace, justice and strong institutions'
```
- [x] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3089 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3089 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3089 
- [Citizen Engagement page](https://www.hackforla.org/citizen-engagement)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_projects/access-the-data.md`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/f2ce8983e77a9b80d455e3677b1cf323092c4f8d/_projects/access-the-data.md?plain=1#L111

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by BeckettOBrien** [comment]
Availability for this week: 6+ Hours
My estimated ETA for completing this issue: Less than 1 Hour

**Comment by tamara-snyder** [comment]
Hi @BeckettOBrien, thanks for picking up this issue. Please move it to the In Progress column on the old Project Board (we aren't using the new one yet) so that no one else will mistakenly assign themselves to it from the Prioritized Backlog.

<img width="333" alt="Screen Shot 2022-06-23 at 3 30 25 PM" src="https://user-images.githubusercontent.com/57715733/175382085-4653c4a6-9d65-4600-98b3-ba9a5b9c1765.png">
